### DISCUSSION TITLE
Reported results for SCG

### DISCUSSION THREAD

**Discussion by daoyuan98** [fredzzhang/upt#24] [Q&A] [answered]
Hi, Thank you for the great work!

In this paper, the reported results for SCG on HICO-DET is 29.26 mAP (full), while the reported result on SCG paper was 31.33 mAP (full). Is the difference caused by different backbones used (i.e. ResNet50-FPN v.s. ResNet101)? Thank you!

**Comment by fredzzhang** [comment, marked as answer]
Hi @daoyuan98,

The `31.33` mAP was obtained using a set of fine-tuned detections provided by the DRG repo, which was later found to be not reproducible, as noted [here](https://github.com/fredzzhang/spatially-conditioned-graphs#hico-det-1). With detections we can reproduce, as noted in the [paper](https://arxiv.org/pdf/2112.01838.pdf), SCG has an mAP of `24.88` using fine-tuned Faster R-CNN, and `29.26` using fine-tuned DETR.

Hope that clarifies things.

Fred.

**Comment by daoyuan98** [comment]
Thank you so much for your information!
